isaaclab_arena/tasks/open_door_task.py

```diff
@@ -5,7 +5,7 @@
 
 import numpy as np
 from dataclasses import MISSING
-
+from typing import Literal
```

Comment:
I think that the formatting is incorrect here.

To fix the formatting, you can run `pre-commit run --all-files` once pre-commit is installed and initialized in the repo.